cassandra/cluster.py

```diff
@@ -85,6 +75,21 @@
 from cassandra.compat import Mapping
 from cassandra.datastax import cloud as dscloud
 
+try:
+    from cassandra.io.twistedreactor import TwistedConnection
+except ImportError:
+    TwistedConnection = type(None)
+
+try:
+    from cassandra.io.eventletreactor import EventletConnection
+except ImportError:
+    EventletConnection = type(None)
```

Comment:
I think I'd prefer we keep that stuff where we use it. The reason is just because reading this is not very clear and we also don't understand why we assign to the None type rather than just None.

Note that I'm not sure if we should really call isubclass() with None types. What about just writing the logic at the place in the code with simple if statements?

Reply:
@aboudreault Howabout now?